snapcraft.yaml

```diff
@@ -19,6 +19,16 @@ apps:
     restart-condition: always
 
 parts:
+  curtin:
+    plugin: python
+    source-type: git
+    source: git://git.launchpad.net/curtin
+    source-branch: ubuntu/devel
```

Comment:
Do we want to use this, or do we want to use the automatic git import of the packages that are in ubuntu? i.e. https://code.launchpad.net/~usd-import-team/ubuntu/+source/curtin/+git/curtin/+ref/applied/ubuntu/bionic-devel ? Cause this git repo will give us what is in bionic-proposed at the time.

Reply:
I think in general we want to use something newer than whatever is in bionic (bionic-updates is 6 months old, although an SRU is currently in progress).

I guess it would be good to ask curtin people (ping @raharper @smoser) what would be a good branch / tag / whatever to use